addons/website/models/ir_http.py

```diff
@@ -379,6 +380,26 @@ def get_frontend_session_info(self):
             })
         return session_info
 
+    @classmethod
+    def _is_allowed_cookie(cls, cookie_type, cookie_name='undefined'):
+        result = super()._is_allowed_cookie(cookie_type, cookie_name)
+
+        if cookie_type not in ["required", "optional"]:
```

Comment:
Why are we using error prone strings when the number of possibilities is strictly enumerated?

Reply:
Changed to: `website` now only handles `'optional'` - other modules can handle their own value(s).